File: NetFramework/DashboardLayout/CS/DashboardLayoutDemo/DemoControls/FlexPie.cs
﻿using C1.Win.Chart;

namespace DashboardLayoutDemo.Controls
{
    public partial class DemoFlexPie : FlexPie, IDemo
    {
        public void Init()
        {
            DataSource = SampleHelper.Instance.GetSourceTable("Sales By Employee", Properties.Resources.SalesByEmployee);
            Binding = "Sales";
            BindingName = "Employee";
        }
    }
}

File: NetFramework/DashboardLayout/VB/DashboardLayoutDemo/DemoControls/FlexPie.vb
Imports C1.Win.Chart

Namespace DashboardDemo.Controls
    Public Partial Class DemoFlexPie
        Inherits FlexPie
        Implements IDemo

        Public Sub Init() Implements IDemo.Init
            DataSource = SampleHelper.Instance.GetSourceTable("Sales By Employee", Resources.SalesByEmployee)
            Binding = "Sales"
            BindingName = "Employee"
        End Sub
    End Class
End Namespace
